setup.py

```diff
@@ -1,16 +1,26 @@
-from setuptools import setup
+from setuptools import setup, find_packages
 
 setup(author='Matthew Egan',
       description='A simple timing utility for long running processes',
       name='howlong',
       py_modules=[
           'HowLong.HowLong',
       ],
+      packages=find_packages(),
       entry_points={
             'console_scripts': [
                   'howlong = HowLong.HowLong:howlong'
             ]
       },
+      install_requires=[
+      'argparse',
+      'logging',
+      'sys',
+      'datetime',
+      'subprocess',
+      'time',
+      'psutil'      
+      ],
```

Comment:
This should only contain `psutil` like so `install_requires=['psutil'],` since all the other requirements are built into the python standard library and we wouldn't want pip to try to [download clones that may be malicious](https://www.bleepingcomputer.com/news/security/ten-malicious-libraries-found-on-pypi-python-package-index/).

Reply:
Hi sorry! Will fix that.